🎨 [Guide Re-org] - Guide landing page revision (#727)
BitcoinDesign/Guide: guide.md

---

<div class="columns-desktop -two">
<div class="column" markdown="1">

<h2 class="h3" markdown="1">[Getting started]({{ '/guide/getting-started/introduction/' | relative_url }})</h2>

New to bitcoin? This section provides top-level overviews of [open design]({{ '/guide/designing-products/open-design/' | relative_url }}), [visual language]({{ '/guide/getting-started/visual-language/' | relative_url }}), [design principles]({{ '/guide/designing-products/principles/' | relative_url }}), a [technology primer]({{ '/guide/getting-started/technology-primer/' | relative_url }}) and more.

</div>
<div class="column" markdown="1">

<h2 class="h3" markdown="1">[Designing bitcoin products]({{ '/guide/designing-products/introduction/' | relative_url }})</h2>

> A closer look at the design process and framworks, from [personal finance use cases]({{ '/guide/designing-products/personal-finance/' | relative_url }}) to [units & symbols]({{ '/guide/payments/units-and-symbols/' | relative_url }}), the [usage life cycle]({{ '/guide/designing-products/usage-life-cycle/' | relative_url }}), [user research]({{ '/guide/designing-products/user-research/' | relative_url }}), and more.

Reviewer: ```suggestion
A closer look at the bitcoin design process, from [personal finance]({{ '/guide/designing-products/personal-finance/' | relative_url }}) use cases and [common user flows]({{ '/guide/designing-products/common-user-flows/' | relative_url }}), to [conducting research]({{ '/guide/designing-products/user-research/' | relative_url }}) and [interoperability]({{ '/guide/designing-products/wallet-interoperability/' | relative_url }}).
```

</div>
</div>

---

<h4 class="h3" markdown="1">Reference designs</h4>

> {% include picture.html

Reviewer: Updating this image to be used as the header image for daily spending wallet, will share it with you so you can use it here when done :)

   image = "/assets/images/guide/daily-spending.png"
   retina = "/assets/images/guide/daily-spending@2x.png"
   imageMobile = "/assets/images/guide/daily-spending-mobile.png"
   retinaMobile = "/assets/images/guide/daily-spending-mobile@2x.png"
   alt-text = "Strong handshake"
   width = 1600
   height = 840
%}

<h2 class="h3" markdown="1">[Daily spending wallet]({{ '/guide/daily-spending-wallet/' | relative_url }})</h2>

> An in-depth exploration of a mobile wallet for a Lightning-first, on-the-go use case. Covers primary user flows like [first use]({{ '/guide/onboarding/first-use/' | relative_url }}), [sending]({{ '/guide/payments/send/' | relative_url }}) and [requesting]({{ '/guide/payments/request/' | relative_url }}), features like [backup]({{ '/guide/onboarding/backing-up-a-wallet/' | relative_url }}) and [contacts]({{ '/guide/payments/contacts/' | relative_url }}), and more.

Reviewer: ```suggestion
An in-depth exploration of a bitcoin-only, non-custodial, mobile lightning-first, wallet for on-the-go payments. Covers primary user flows like [first use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}), [sending]({{ '/guide/daily-spending-wallet/sending/' | relative_url }}) and [requesting]({{ '/guide/daily-spending-wallet/requesting/' | relative_url }}), features like [backup]({{ '/guide/daily-spending-wallet/backup-and-recovery/' | relative_url }}) and [contacts]({{ '/guide/daily-spending-wallet/contacts/' | relative_url }}), and more.
```
Just wanted to make the use-case a bit more explicit

Author: I think it's too much. Very hard to read this way.

   alt-text = "Strong handshake"
   width = 400
   height = 210
%}

<h2 class="h3" markdown="1">[Upgradeable account]({{ '/guide/case-studies/upgradeable-wallet/' | relative_url }})</h2>

Exploring the concept of progressive security, in which applications help users navigate varying security configurations appropriate to their holdings and use case.

</div>
</div>

<div class="columns-desktop -two -ref">
<div class="column" markdown="1">

{% include picture.html
   image = "/assets/images/guide/shared-accounts.png"
   retina = "/assets/images/guide/shared-accounts@2x.png"
   alt-text = "Strong handshake"
   width = 400
   height = 210
%}

<h2 class="h3" markdown="1">[Shared accounts]({{ '/guide/case-studies/shared-account/' | relative_url }})</h2>

An overview and considerations for bitcoin wallets that are managed by multiple owners.

</div>
<div class="column -blank" markdown="1">

</div>
</div>

---

<h2 class="h3" markdown="1">[How it works]({{ '/how-it-works/' | relative_url }})</h2>

Explanations of various technical aspects of bitcoin and Lightning.

<div class="columns-desktop -two -how">
<div class="column" markdown="1">

> [Private key management]({{ '/guide/private-key-management/introduction/' | relative_url }})<br />
> [Automatic cloud backup]({{ '/guide/private-key-management/cloud-backup/' | relative_url }})<br />
> [Manual backup]({{ '/guide/private-key-management/manual-backup/' | relative_url }})<br />
> [External signers]({{ '/guide/private-key-management/external-signers/' | relative_url }})<br />
> [Multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }})

Reviewer: ```suggestion
[Private key management]({{ '/guide/how-it-works/private-key-management/introduction/' | relative_url }})<br />
[Automatic cloud backup]({{ '/guide/how-it-works/private-key-management/cloud-backup/' | relative_url }})<br />
[Manual backup]({{ '/guide/how-it-works/private-key-management/manual-backup/' | relative_url }})<br />
[External signers]({{ '/guide/how-it-works/private-key-management/external-signers/' | relative_url }})<br />
[Multi-key]({{ '/guide/how-it-works/private-key-management/multi-key/' | relative_url }})
```

Updated links

Author: Link changes should be part of the update from master. Otherwise I'd be merging in URLs that don't exist in the branch yet.

</div>
<div class="column" markdown="1">

> [Bitcoin backups]({{ '/guide/private-key-management/backups/' | relative_url }})<br />
> [Payment request formats]({{ '/guide/payments/request/payment-request-formats/' | relative_url }})<br />
> [Coin selection]({{ '/guide/glossary/coin-selection/' | relative_url }})<br />
> [Nodes]({{ '/guide/glossary/node/' | relative_url }})<br />
> [Transaction lifecycle]({{ '/guide/payments/transactions/' | relative_url }})

Reviewer: ```suggestion
[Bitcoin backups]({{ '/guide/how-it-works/backups/' | relative_url }})<br />
[Payment request formats]({{ '/guide/how-it-works/payment-request-formats/' | relative_url }})<br />
[Coin selection]({{ '/guide/how-it-works/coin-selection/' | relative_url }})<br />
[Nodes]({{ '/guide/how-it-works/nodes/' | relative_url }})<br />
[Transactions]({{ '/guide/how-it-works/transactions/' | relative_url }})
```

Updated links

</div>
</div>

---

<h2 class="h3" markdown="1">[Resources]({{ '/guide/resources/' | relative_url }})</h2>

> [Research references]({{ '/guide/designing-products/user-research/resources' | relative_url }}), [shared design files]({{ '/guide/designing-products/design-resources/' | relative_url }}), and other useful assets.

Reviewer: ```suggestion
[Design files]({{ '/guide/resources/design-files/' | relative_url }}), [design ressearch]({{ '/guide/resources/design-research/' | relative_url }}), and other useful assets.
```

Updated links

<div class="columns-desktop -two -ref">
<div class="column" markdown="1">

{% include picture.html
   image = "/assets/images/guide/savings-account.png"
   retina = "/assets/images/guide/savings-account@2x.png"
   alt-text = "Strong handshake"
   width = 400
   height = 210
%}

<h2 class="h3" markdown="1">[Savings wallet]({{ '/guide/case-studies/savings-account/' | relative_url }})</h2>

A multi-key mobile wallet design that provides great security for larger amounts, and low-friction for smaller transactions.

</div>
<div class="column" markdown="1">

{% include picture.html
   image = "/assets/images/guide/upgradeable-account.png"
   retina = "/assets/images/guide/upgradeable-account@2x.png"
   alt-text = "Strong handshake"
   width = 400
   height = 210
%}

> <h2 class="h3" markdown="1">[Upgradeable account]({{ '/guide/case-studies/upgradeable-wallet/' | relative_url }})</h2>

Reviewer: We should call this upgradeable wallet to fit the other reference designs.

Author: Separate PR.

Exploring the concept of progressive security, in which applications help users navigate varying security configurations appropriate to their holdings and use case.

</div>
</div>

<div class="columns-desktop -two -ref">
<div class="column" markdown="1">

{% include picture.html
   image = "/assets/images/guide/shared-accounts.png"
   retina = "/assets/images/guide/shared-accounts@2x.png"
   alt-text = "Strong handshake"
   width = 400
   height = 210
%}

> <h2 class="h3" markdown="1">[Shared accounts]({{ '/guide/case-studies/shared-account/' | relative_url }})</h2>

Author: Separate PR.
